### ISSUE TITLE
about geonerf

### ISSUE THREAD

**Issue by Fangkang515** [medakk/geonerf#1]
Hello, I want to ask, is this code the implementation of “GeoNeRF: Generalizing NeRF with Geometry Priors”？

Thanks!

**Comment by medakk** [comment]
No, this code is unrelated to that project. This repo is an idea I was playing with to try to compute geodesics on a mesh using priors from NeRF training
